GraphAlgorithmsTests/MCSNaiveFinderTest.cs

```diff
@@ -0,0 +1,126 @@
+﻿using NUnit.Framework;
+
+namespace GraphAlgorithms.Tests
+{
+    [TestFixture]
+    public class MCSNaiveFinderTest
+    {
+        private static IEnumerable<TestCaseData> TestCases()
+        {
+            Graph graph1;
+            Graph graph2;
+            HashSet<int[]> subgraphs1; // sets in case someone would like to be very diligent & actually enumerate all possible MCS's (but I'm not doing that here)
+            HashSet<int[]> subgraphs2;
+
+            // two completely different graphs
+            graph1 = new Graph(3);
+            graph1.AddBothSidesEdge(0, 1);
+            graph2 = new Graph(3);
+            subgraphs1 = new HashSet<int[]>() { new int[] { 0, 2 } };
+            subgraphs2 = new HashSet<int[]>() { new int[] { 0, 1 } };
```

Comment:
here the solution could also be {0, 2}, {0, 2} so the test will fail for a correct answer

Reply:
I wrote a comment above that I'm using a hash set to potentially also include other solutions. But since the algorithm is deterministic I think we just can check for those special cases